src/components/AdminNavbar/AdminNavbar.tsx

```diff
@@ -33,10 +37,20 @@ function AdminNavbar({ targets, url_1 }: NavbarProps): JSX.Element {
   const [spamCountData, setSpamCountData] = useState([]);
 
   const currentUrl = window.location.href.split('=')[1];
+  const userID = localStorage.getItem('id');
 
-  const { data, error, refetch } = useQuery(ORGANIZATIONS_LIST, {
+  const { data: user_data } = useQuery(USER_ORGANIZATION_LIST, {
+    variables: { id: userID },
+  });
+
+  const { data: organization_data } = useQuery(ORGANIZATIONS_LIST, {
     variables: { id: currentUrl },
   });
+
+  const user = user_data?.user;
+  const organization = organization_data?.organizations[0];
+  const OrgName = organization?.name;
+
```

Comment:
Use loading and error variables inside the queries, otherwise tests would give warnings and errors. Also make sure to use if(!data){...} thing.